pyproject.toml

```diff
@@ -76,7 +76,7 @@ select = [ "D", "E", "F", "W", "RUF",]
 [tool.poetry.dependencies]
 python = ">=3.8,<3.11"
 coloredlogs = ">=10,<16"
-sanic = "^21.12.0"
+sanic = ">=22.9.1,<22.12"
```

Comment:
I would double-check what's the latest LTS of sanic we could upgrade to, I can't remember if `22.0.0` is one of them.

Reply:
You're right.
Our LTS [options](https://sanic.dev/en/organization/policies.html#long-term-support-v-interim-releases) are 23.12 or 22.12

Reply:
23.12 seems released on 31-Dec, could be coming with undetected bugs? I'll let you decide which version is best.

Reply:
Yeah.
We have till Dec to use 22.12, so I'll use that instead.